drivers/sensor/adxl362/adxl362.c

```diff
@@ -468,7 +468,7 @@ static int adxl362_channel_get(struct device *dev,
 		val->val1 = data->acc_z / 1000;
 		val->val2 = (data->acc_z % 1000) * 1000;
 		break;
-	case SENSOR_CHAN_TEMP: /* Temperature in degrees Celsius. */
+	case SENSOR_CHAN_AMBIENT_TEMP: /* Temperature in degrees Celsius. */
```

Comment:
I believe, this should be SENSOR_CHAN_DIE_TEMP. According to the datasheet:
"The ADXL362 includes an integrated temperature sensor that can monitor internal system temperature or improve the temperature stability of the device via calibration. For example, acceleration outputs vary with temperature at a rate of ±0.5 mg/°C (typical), but the relationship to temperature is repeatable and can be calibrated."

Reply:
Yes, correct! I can change it and re-push the commit.
By-the way I now re-checked all the drivers in this way: whenever a TEMP sensor was
inside a XYZ sensor (accel/gyro/mag type) then I downloaded the datasheet and checked it
was a DIE_TEMP sensor. I came to the conclusion that also the following must be changed:

- bma280
- bmg160
- bmi160
- fxos8700
- mpu6050

Moreover, for bmg160/bmi160/fxos8700 I will change the related samples file accordingly (DIE_TEMP instead of AMBIENT_TEMP).

Does it sound good to you?